api_config.toml

```diff
@@ -1,5 +1,9 @@
 # This is the default toml config file that is used by the rest API server.
```

Comment:
i think since we have define another `api_config.toml` in freva-rest let's remove this, yes?

Reply:
Not really, this is used for dev purposes only. If you want me to move it somewhere then to the `dev-env` folder.